lib/std/net/test.zig

```diff
@@ -296,6 +296,25 @@ test "listen on a unix socket, send bytes, receive bytes" {
     try testing.expectEqualSlices(u8, "Hello world!", buf[0..n]);
 }
 
+// On Linux, set REUSEPORT option on an unix socket returns a EOPNOTSUPP error.
+// see https://github.com/torvalds/linux/commit/5b0af621c3f6ef9261cf6067812f2fd9943acb4b
+test "listen on a unix socket with reuse_port option" {
+    if (!net.has_unix_sockets) return error.SkipZigTest;
+    if (builtin.os.tag != .linux and comptime !builtin.os.tag.isDarwin()) {
+        // TODO build abstractions for other operating systems
+        return error.SkipZigTest;
+    }
```

Comment:
Why not just check for Windows specifically? This check will exclude other POSIX platforms.

Also, see #363; either file an enhancement issue and link it here or remove the TODO and just explain why the test is skipped.